dali/operators/color/brightness_contrast.cc

```diff
@@ -13,30 +13,33 @@
 // limitations under the License.
 
 #include "dali/operators/color/brightness_contrast.h"
-#include "dali/kernels/imgproc/color_manipulation/brightness_contrast.h"
+#include "dali/kernels/imgproc/pointwise/multiply_add.h"
 
 namespace dali {
 namespace {
 
 template <typename Out, typename In>
-using TheKernel = kernels::BrightnessContrastCpu<Out, In, 3>;
+using TheKernel = kernels::MultiplyAddCpu<Out, In, 3>;
 
 }  // namespace
 
 
 DALI_SCHEMA(BrightnessContrast)
-                .DocStr(R"code(Change the brightness and contrast of the image.
+    .DocStr(R"code(Change the brightness and contrast of the image.
```

Comment:
I would add more description how bellow arguments are used to transform pixel values.